doc/language_guide.md

```diff
@@ -305,38 +305,93 @@ Example:
 */
 ```
 
-## Models
+## Prediction syntax
+
+A SQLFlow prediction statement consists of a sequence of select, predict, and using clauses.
+
+```
+SELECT select_expr [, select_expr ...]
+FROM table_references
+  [WHERE where_condition]
+  [LIMIT row_count]
+PREDICT result_table_reference
+[WITH
+  attr_expr [, attr_expr ...]]
+USING model_table_reference;
+```
 
-SQLFlow supports various TensorFlow premade estimators.
+### Select clause
 
-### DNNClassifer
+The [select clause](#select-clause) syntax is the same as the select clause syntax in the training syntax. SQLFlow uses the column name to guarantee the prediction data has the same order as the training data. For example, if we have used `c1`, `c2`, `c3` and `label` column to train a model, the select clause in the prediction job should also retrieve columns that contain exactly the same names.
+
+### Predict and using clause
```

Comment:
Predict and Using Clause

title capitalizing rules: https://wenku.baidu.com/view/07e52c69a98271fe910ef9f4.html

Reply:
Should I update all section titles? For example, "Select clause" => "Select Clause", "Prediction syntax" => "Prediction Syntax".

Reply:
We should, and seems that some of the documents also need update like `run_with_hive.md`